### ISSUE TITLE
Unexpected sorting results 

### ISSUE THREAD

**Issue by TheMickael** [Cologler/logseq-sort-blocks-typescript#2]
First of all thanks for making a sorting plugin 👍

From what I understand your algorithm sort block by block, including markup symbols like '#' and '[[' which will show an unintuitive result.

The below example is sorted A-Z

- [[BB]]
- [[AA]]
- #A
- #B
- A
- B

Would be great to get an update 👍

**Comment by Cologler** [comment]
The currently sorted result is

![image](https://github.com/Cologler/logseq-sort-blocks-typescript/assets/10906962/22641658-7dd0-44ec-87aa-60956ab72a40)

Look good to me, based on my theme.

**Comment by TheMickael** [comment]
Yes that is the result, which is OK if you want your blocks sorted first by type and then alphabetically, which could be what you want.
It becomes difficult when lets say you want to sort peoples names and some of them have a a page, some a tag and some nothing. 

I think there are valid cases for both, so maybe have an option or be clear in the description how it sorts?


Cheers



/Mickael

**Comment by Cologler** [comment]
I am okay with some options for this.

But removing markdown markup before sorting is not easy:

- `#a` -> `a`
- `# a` -> `# a`
- `[[a]]` -> `a`
- `[[a]] #a` -> `a a`
- `[[a]] # a` -> `a # a`
- `#[[a]]` -> `a`
- `[a](https://github.com/sss#b)` -> `a`
- `[#a](https://github.com/sss#b)` -> `(https://github.com/sssb)` WTF

It seems like I have to write an entire markdown parser for logseq.
